### DISCUSSION TITLE
What is Effective 30-day limit in API v2?

### DISCUSSION THREAD

**Discussion by legendaryangelist** [tweepy/tweepy#2128] [Questions]
Hello @Harmon758.
Thanks for your kindness.
I have another question.
I saw **Effective 30-day limit** in the doc related to rate limit.
So rate limit of **GET_2_tweets_search_recent** endpoint is 60 requests / 15 min per app.
But **Effective 30-day limit** of **GET_2_tweets_search_recent** endpoint is 10k.
What does it mean?
So I can't use **GET_2_tweets_search_recent** endpoint more than 10k every month?
Because one day is 1440 min. So I can request up to 60 * 1440 / 15 = 5760 every day.
Sorry ))
Please explain.

**Comment by Harmon758** [comment]
Yes, it likely refers to a sliding window rate limit for the last 30 days, rather than a rate limit with fixed times.

This doesn't really have anything to do with Tweepy, and I don't speak for Twitter or have any insight into what they mean, so this question would be more appropriate for the [Twitter Developers Forums](https://twittercommunity.com/), although you're unlikely to get an official response there these days.
